src/components/Textarea/Textarea.css

```diff
@@ -5,3 +5,7 @@
 .form-textarea--error {
   @apply border-red-500;
 }
+
+.form-textarea--borderless {
+  @apply border-0;
```

Comment:
@i-pip should perhaps be `border-none` it keeps the same width of the border compared to other inputs.

Reply:
Thanks for the review @MildTomato! Didn't know about `border-none` - very cool! I've pushed the commit, thanks for the other changes too.